packages/model/history.go

```diff
@@ -0,0 +1,49 @@
+package model
+
+import (
+	"errors"
+	"fmt"
+	"time"
+)
+
+const keysUpdatePattern = `UPDATE "1_keys" SET amount = amount + ? WHERE id = ?`
+
+var errLowBalance = errors.New("not enough APL on the balance")
+
+// History represent record of history table
+type History struct {
+	ID          int64
+	SenderID    int64
+	RecipientID int64
+	Amount      float64
+	Comment     string
+	BlockID     int64
+	TxHash      []byte
+	CreatedAt   time.Time
+}
+
+// TokenTransferWithHistory add one history record and update wallet balance
+func TokenTransferWithHistory(transaction *DbTransaction, history History) error {
```

Comment:
Where this function used? We are decided, that it will be used in contracts to transfer tokens, MoneyTransfer contract is still the same, u should add it to contract functions and use in MoneyTransfer